stablehlo/dialect/StablehloOps.td

```diff
@@ -1471,6 +1471,24 @@ def StableHLO_SliceOp: StableHLO_Op<
       [NoSideEffect, SameOperandsAndResultElementType,
        AllTypesMatch<["start_indices", "limit_indices", "strides"]>,
        DeclareOpInterfaceMethods<InferTypeOpInterface>]> {
+  let summary = "Iota operator";
+  let description = [{
+    Extracts a sub-tensor from the `operand` using the dimension indices from
+    the operands.
+
+    See:
+    https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md#stablehloslice
+
+    Example:
+    ```mlir
+    %0 = "stablehlo.slice" (%arg0) {
+      start_indices = dense<2> : tensor<i64>,
```

Comment:
This should be `tensor<1xi64>` and same for other attributes below.

Reply:
You're right. Thanks.